### PR TITLE
Clean up existing instances when updating the platform view

### PR DESCRIPTION
**Pull request by bwikbs**
Resolved following issues:
https://github.com/flutter-tizen/plugins/issues/132

- Fixed the problem that platform view instance was not updated properly when restarted

### CODE REVIEW

File: shell/platform/tizen/channels/platform_view_channel.cc [hunk 1]
```diff
@@ -135,6 +135,12 @@ void PlatformViewChannel::HandleMethodCall(
         focused_view->second->SetFocus(false);
       }
 
+      auto existing_view = view_instances_.find(view_id);
+      if (existing_view != view_instances_.end()) {
+        existing_view->second->Dispose();
+        view_instances_.erase(existing_view);
+      }
+
```
Comment (swift-kim) [inline shell/platform/tizen/channels/platform_view_channel.cc:143]: Here doesn't seem a right place to dispose view instances. Only a single view instance can exist per app in this way.

File: shell/platform/tizen/flutter_tizen_engine.h [hunk 2]
```diff
@@ -70,6 +70,8 @@ class FlutterTizenEngine : public TizenRenderer::Delegate {
   bool RunEngine(const FlutterDesktopEngineProperties& engine_properties);
   bool StopEngine();
 
+  void RestartEngine();
```
Comment (bbrto21) [inline shell/platform/tizen/flutter_tizen_engine.h:73]: I think RestartEngine is an ambiguous name. First of all, there is no API called start, and This implementation does nothing to the FlutterEngine instance.

Comment (bwikbs) [reply shell/platform/tizen/flutter_tizen_engine.h:73]: Hmm..Then.. How about ``reset``?

File: shell/platform/tizen/flutter_tizen_engine.cc [hunk 3]
```diff
@@ -194,6 +194,12 @@ bool FlutterTizenEngine::RunEngine(
         auto engine = reinterpret_cast<FlutterTizenEngine*>(user_data);
         auto message = engine->ConvertToDesktopMessage(*engine_message);
         engine->message_dispatcher->HandleMessage(message);
+
+        // Workarounds for handling engine restarts. Needs to be replaced by
+        // PlatformView::OnPreEngineRestart.
+        if (strncmp(message.channel, "flutter/isolate", 15) == 0) {
+          engine->RestartEngine();
+        }
```
Comment (bbrto21) [inline shell/platform/tizen/flutter_tizen_engine.cc:202]: What is the exact purpose of the "flutter/isolate" channel? 
```FlutterDesktopMessage``` also has a channel and message. I think you should check the message too.
Lastly, Is this workaround a necessary?

Comment (bwikbs) [reply shell/platform/tizen/flutter_tizen_engine.cc:202]: Message contains isolate's id like this:
```
isolates/880563550613439
```
Now that I see it, this workaround can't use this propose. So I'll revert it.

File: shell/platform/tizen/flutter_tizen_engine.h (hunk 2 above)
Comment (bwikbs) [reply shell/platform/tizen/flutter_tizen_engine.h]: It's reverted

**Review by bbrto21 — APPROVED** [review]

**Pull request merged**